x/docs/md/cli/linux.md

```diff
@@ -70,5 +72,15 @@ COMPLETION_SECTION
 ### Continue
 You can now continue by [choosing a language](http://exercism.io/languages).
 
+### Uninstalling the Exercism CLI
+
+Remove the files from the `~/bin` directory with the following command
```

Comment:
This is a singular "file" rather than the plural "files" as shown.  It could do with a colon or semicolon at the end of the line, perhaps.